packages/rrweb/src/replay/timer.ts

```diff
@@ -10,64 +10,71 @@ export class Timer {
   public speed: number;
 
   private actions: actionWithDelay[];
-  private raf: number | null = null;
-  private liveMode: boolean;
+  private raf: number | boolean | null = null;
```

Comment:
```suggestion
  private raf: number | true | null = null;
```
I think we can make this type more precise.